client/app/telecom/pack/resiliation/pack-resiliation.less

```diff
@@ -1,28 +1,16 @@
 .telecom-pack-resiliation {
 
-    .service-type {
-        margin-bottom: 16px;
-
-        .service-item {
-            .service-select-all {
-                background-color: @gray-light;
-            }
-
-            .service-select-single {
-                padding: 4px 0;
+    .service-select-all {
+        background-color: @gray-light;
```

Comment:
Please use `@gray-lighter` in this case.

Reply:
Suggestion : maybe we should use bg-* classes instead of declare some less rules ?